src/os/ObjectStore.cc

```diff
@@ -21,7 +21,7 @@
 #if defined(WITH_BLUESTORE)
 #include "bluestore/BlueStore.h"
 #endif
-#ifndef WITH_SEASTAR
+#ifdef WITH_KSTORE
```

Comment:
I think it's a good idea to introduce conditional building of `KStore` also in classical. It would enable us to turn it off by default and, someday, drop completely, I hope.